src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -14,27 +16,102 @@ namespace System.Net.Security.Tests
 {
     public class SslStreamStreamToStreamTest
     {
+        private readonly byte[] sampleMsg = Encoding.UTF8.GetBytes("Sample Test Message");
+        private const int TaskWaitTime = 15; // seconds
+
+        #region Tests
```

Comment:
I used to think the same way. See the following article presenting some interesting points on why regions are always bad:
   * http://programmers.stackexchange.com/questions/53086/are-regions-an-antipattern-or-code-smell
   * StyleCop rule# SA1124: DoNotUseRegions